### PR TITLE
chore: fix CI test

### PR DESCRIPTION
**Pull request by sarfata**

### CODE REVIEW

File: puncover/gcc_tools.py [hunk 1]
```diff
@@ -15,12 +15,16 @@ def __init__(self, gcc_base_filename):
     def gcc_tool_path(self, name):
         path = self.gcc_base_filename + name
         if not os.path.isfile(path):
-            raise Exception("Could not find %s" % path)
+            # try prepending 'gcc-' - useful for nm on some gcc distributions
+            path = self.gcc_base_filename + "gcc-" + name
+            if not os.path.isfile(path):
+                raise Exception("Could not find %s" % path)
```
Comment (HBehrens) [inline puncover/gcc_tools.py:21]: would be nice to have a test for this

**Comment by HBehrens** [comment]
Even though I wonder if building and downloading tool chains is the most-effective way of doing this, fixing the setup in places is probably less work than revamping this entirely.

@sarfata Are you still planning to work on this? I for one see quite some value in doing an integration test.

**Comment by HBehrens** [comment]
ping @sarfata

**Comment by sarfata** [comment]
> @sarfata Are you still planning to work on this? I for one see quite some value in doing an integration test.

not in the near future.

**Comment by HBehrens** [comment]
Let me close it then in the name of house keeping. We can always restore the branch if necessary. Thank you!

**Pull request closed without merging**